code/datums/craft/recipes/machinery.dm

```diff
@@ -25,24 +25,43 @@
 
 /datum/craft_recipe/machinery/modularconsole
 	name = "modular console frame"
-	result = /obj/machinery/modular_computer/console/buildable
+	result = /obj/item/modular_computer/console
```

Comment:
Why is a console treated as an obj/item ? Are there measures to stop someone picking it up or wearing it?

Reply:
No idea, all claims to actual maker of them. We can rework it later tho

Reply:
i'm sure theres a reason, but make sure you find out the answer to these questions

Reply:
that answers the first question, now what about the second?
What measures are taken to prevent consoles/telescreens from being used in ways they shouldnt, like being picked up, worn, thrown, inserted into disposals, etc

Reply:
They are anchored